sdk/bpf/scripts/install.sh

```diff
@@ -178,7 +178,9 @@ if [[ ! -e rust-bpf-$machine-$version.md || ! -e rust-bpf-$machine ]]; then
 fi
 
 # Install Rust-BPF Sysroot sources
-version=v0.14
+rm -rf rust-bpf-sysroot*
+rm -rf ~/.cache/solana/v1.0
```

Comment:
it seems this causes the sysroot to be cloned and rebuilt on the build of every program in `programs/rust/*`, which dramatically slows down testing.